static/js/modules/typeahead.js

```diff
@@ -116,9 +116,7 @@ module.exports = {
       window.location = window.location.origin + '/' + datasetName + '/' + datum.id;
     });
 
-    $('.twitter-typeahead').css({
-        display: 'flex',
-    });
+    $('.twitter-typeahead').addClass('flex-container').css('display','');
```

Comment:
FYI need to a class to take advantage of Bourbon's browser prefixing for the `display` property. And then the css method is to override the fact that the library sets an inline style of `display: inline-block`.